content/help/ops/traffic-management/proxy-cmd/index.md

```diff
@@ -278,6 +278,70 @@ one route that matches on everything. This route tells Envoy to send the request
         }
     ]
     {{< /text >}}
+    
+1. When envoy use the `serviceName` field as a key that have got the list of Endpoints and then will proxy the request to one of them.
```

Comment:
To see the endpoints currently available for the cluster use the `proxy-config endpoints` command.

Reply:
Done.